tika-core/src/main/java/org/apache/tika/parser/Parser.java

```diff
@@ -68,4 +67,31 @@ void parse(
             Metadata metadata, ParseContext context)
             throws IOException, SAXException, TikaException;
 
+    /**
```

Comment:
this method is not needed. We shouldn't be changing the Parser's public interface. If you need to pass a flag parameter, then pass it through the ParseContext.